# Type: feat
# Subtype: Ex
# English Summary: Added getCommitPath() function to obtain the path of the temporary block
# Chinese Summary: 添加了getCommitPath()函数来获取临时块的路径
# Entry Number: 17

# === Git Diff Content ===

diff --git a/servers/src/main/java/tachyon/worker/block/meta/TempBlockMeta.java b/servers/src/main/java/tachyon/worker/block/meta/TempBlockMeta.java
index 075574e..2a7873f 100644
--- a/servers/src/main/java/tachyon/worker/block/meta/TempBlockMeta.java
+++ b/servers/src/main/java/tachyon/worker/block/meta/TempBlockMeta.java
@@ -4,9 +4,9 @@
  * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
  * "License"); you may not use this file except in compliance with the License. You may obtain a
  * copy of the License at
- * 
+ *
  * http://www.apache.org/licenses/LICENSE-2.0
- * 
+ *
  * Unless required by applicable law or agreed to in writing, software distributed under the License
  * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
  * or implied. See the License for the specific language governing permissions and limitations under
@@ -33,6 +33,10 @@ public class TempBlockMeta extends BlockMetaBase {
     return CommonUtils.concatPath(mDir.getDirPath(), mUserId, mBlockId);
   }
 
+  public String getCommitPath() {
+    return CommonUtils.concatPath(mDir.getDirPath(), mBlockId);
+  }
+
   public void setBlockSize(long newSize) {
     mBlockSize = newSize;
   }
